opsdroid/database/matrix/__init__.py

```diff
@@ -165,19 +164,22 @@ async def get(self, key, get_full=False):
         )
 
         ori_data = await self.connector.connection.room_get_state_event(
-            room_id=self.room_id,
-            event_type=self._event_type,
-            state_key=state_key,
+            room_id=self.room_id, event_type=self._event_type, state_key=state_key
         )
+
         if isinstance(ori_data, RoomGetStateEventError):
-            raise RuntimeError(
-                f"Error getting {key} from matrix room {self.room_id}: {ori_data.message}({ori_data.status_code})"
-            )
-        elif ori_data.transport_response.status == 404:
-            _LOGGER.error(
-                f"Error getting {key} from matrix room {self.room_id}: Event not found"
-            )
-            return
+            if (
+                ori_data.transport_response is not None
+                and ori_data.transport_response.status == 404
+            ):
+                _LOGGER.error(
+                    f"Error getting {key} from matrix room {self.room_id}: Event not found"
+                )
+                return
+            else:
```

Comment:
I assume that your linter has told you this and you've very reasonably ignored it but `else` is redundant after `return`.